Analysis/Utkscan/core/source/DetectorDriverXmlParser.cpp

```diff
@@ -131,7 +131,11 @@ vector<EventProcessor*> DetectorDriverXmlParser::ParseProcessors(
         } else if (name == "DoubleBetaProcessor") {
             vecProcess.push_back(new DoubleBetaProcessor());
         } else if (name == "PspmtProcessor") {
-            vecProcess.push_back(new PspmtProcessor());
+            vecProcess.push_back(new PspmtProcessor(
+                    processor.attribute("vd").as_string(),
```

Comment:
You should probably have this provide a default value. I suggest defaulting to the version used in the experiment.